docs/src/app/(public)/(content)/react/utils/use-render/demos/render/css-modules/index.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+import * as React from 'react';
+import { useRender } from '@base-ui-components/react/use-render';
+import styles from './index.module.css';
+
+type TextProps = {
+  render?: useRender.RenderProp<Record<string, unknown>>;
+  children: React.ReactNode;
+};
+
+function Text(props: TextProps) {
+  const { render, ...otherProps } = props;
```

Comment:
Default assignment `render = <p />` seems like a cleaner example to use throughout the docs? (Same for `button` ones)

```jsx
function Text(props: TextProps) {
  const { render = <p />, ...otherProps } = props;

  const { renderElement } = useRender({
    render,
    props: { ...otherProps, className: styles.Text },
  });
  // ...
}
```

Reply:
Done.